core/utils/utils.go

```diff
@@ -125,29 +81,6 @@ func NewSecret(n int) string {
 	return base64.StdEncoding.EncodeToString(b)
 }
 
-// RemoveHexPrefix removes the prefix (0x) of a given hex string.
-func RemoveHexPrefix(str string) string {
-	if HasHexPrefix(str) {
-		return str[2:]
-	}
-	return str
-}
-
-// HasHexPrefix returns true if the string starts with 0x.
-func HasHexPrefix(str string) bool {
-	return len(str) >= 2 && str[0] == '0' && (str[1] == 'x' || str[1] == 'X')
-}
-
-// IsEmptyAddress checks that the address is empty, synonymous with the zero
-// account/address. No logs can come from this address, as there is no contract
-// present there.
-//
-// See https://stackoverflow.com/questions/48219716/what-is-address0-in-solidity
-// for the more info on the zero address.
-func IsEmptyAddress(addr common.Address) bool {
-	return addr == ZeroAddress
-}
-
 // StringToHex converts a standard string to a hex encoded string.
 func StringToHex(in string) string {
 	return AddHexPrefix(hex.EncodeToString([]byte(in)))
```

Comment:
I can't add the comment lower but I noticed we moved the `BackoffSleeper` to `core/chains/evm/utils`. Should we remove it from below?